_components/range-slider/guidance/implementation.md

```diff
@@ -1 +1,3 @@
-- Set the `min` and `max` attribute of the `input` element to correspond to the instructions or labels that accompany the slider.
\ No newline at end of file
+- Set the `min` and `max` attribute of the `input` element to correspond to the instructions or labels that accompany the slider.
+- Set the `data-text-unit` attribute of the `input` element to add a unit to screen reader callouts. The unit will be read after the current value.
+- Set the `data-text-preposition` attribute of the `input` element to change the preposition between current value and max value. If no value is present, it will default to `of`. When providing a range slider on pages in a language other than English, be sure to update as necessary.
```

Comment:
Wondering if we can move these to a properties table like on the [in-page nav](https://designsystem.digital.gov/components/in-page-navigation/#in-page-navigation-initialization-properties) or [date picker](https://designsystem.digital.gov/components/date-picker/#using-the-date-picker-component) pages.

Note: We are not 100% consistent on how we document `data-` attributes (see [tooltip page](https://designsystem.digital.gov/components/tooltip/#using-the-tooltip-component)), but I find the table to be clearer and the better pattern.

Reply:
I prefer the table as well! I'll add it shortly! 👍

Reply:
I went ahead and moved them to a table like the examples provided! I removed the blurbs about them from above. Do you think I should include `min` and `max` as well?

Resolved in [4426a54](https://github.com/uswds/uswds-site/pull/2310/commits/4426a54872e89bf32fbae4aa04b9246518e2484b)